docs/querying.md

```diff
@@ -160,54 +165,139 @@ as well.
+[Op.contains]: 2           // @> '2'::integer (PG range contains element operator)
+[Op.contains]: [1, 2]      // @> [1, 2) (PG range contains range operator)
+[Op.contained]: [1, 2]     // <@ [1, 2) (PG range is contained by operator)
+[Op.overlap]: [1, 2]       // && [1, 2) (PG range overlap (have points in common) operator)
+[Op.adjacent]: [1, 2]      // -|- [1, 2) (PG range is adjacent to operator)
+[Op.strictLeft]: [1, 2]    // << [1, 2) (PG range strictly left of operator)
+[Op.strictRight]: [1, 2]   // >> [1, 2) (PG range strictly right of operator)
+[Op.noExtendRight]: [1, 2] // &< [1, 2) (PG range does not extend to the right of operator)
+[Op.noExtendLeft]: [1, 2]  // &> [1, 2) (PG range does not extend to the left of operator)
 ```
 
-### Combinations
+#### Combinations
 ```js
+const Op = Sequelize.Op;
+
 {
   rank: {
-    $or: {
-      $lt: 1000,
-      $eq: null
+    [Op.or]: {
+      [Op.lt]: 1000,
+      [Op.eq]: null
     }
   }
 }
 // rank < 1000 OR rank IS NULL
 
 {
   createdAt: {
-    $lt: new Date(),
-    $gt: new Date(new Date() - 24 * 60 * 60 * 1000)
+    [Op.lt]: new Date(),
+    [Op.gt]: new Date(new Date() - 24 * 60 * 60 * 1000)
   }
 }
 // createdAt < [timestamp] AND createdAt > [timestamp]
 
 {
-  $or: [
+  [Op.or]: [
     {
       title: {
-        $like: 'Boat%'
+        [Op.like]: 'Boat%'
       }
     },
     {
       description: {
-        $like: '%boat%'
+        [Op.like]: '%boat%'
       }
     }
   ]
 }
 // title LIKE 'Boat%' OR description LIKE '%boat%'
 ```
 
+#### Operators Aliases
+Sequelize allows setting specific strings as aliases for operators -
+```js
+const Op = Sequelize.Op;
+const operators = {
+  $gt: Op.gt
+}
+const connection = new Sequelize(db, user, pass, { operators })
+
+[Op.gt]: 6 // > 6
+$gt: 6 // same as using Op.gt (> 6)
+```
+
+
+#### Operators security
+Using Sequelize without any aliases improves security.
+Some frameworks automatically parse user input into js objects and if you fail to sanitize your input it might be possible to inject an Object with string operators to Sequelize.
+
+Not having any string aliases will make it extremely unlikely that operators could be injected but you should always properly validate and sanitize user input.
+
+For backward compatibility reasons Sequelize sets the following aliases by default -
+$eq, $ne, $gte, $gt, $lte, $lt, $not, $in, $notIn, $is, $like, $notLike, $iLike, $notILike, $regexp, $notRegexp, $iRegexp, $notIRegexp, $between, $notBetween, $overlap, $contains, $contained, $adjacent, $strictLeft, $strictRight, $noExtendRight, $noExtendLeft, $and, $or, $any, $all, $values, $col
+
+currently the following legacy aliases are also set but are planned to be fully removed in the near future -
+ne, not, in, notIn, gte, gt, lte, lt, like, ilike, $ilike, nlike, $notlike, notilike, .., between, !.., notbetween, nbetween, overlap, &&, @>, <@
+
+Due to the security improvements it is highly advised to only use the specific aliases you need.
```

Comment:
For better security it is highly advised to use `Sequelize.Op` and not depend on any string alias at all. You can limit alias your application will need by setting `operators` option, remember to sanitize user input especially when you are directly passing them to Sequelize methods.